### DISCUSSION TITLE
i lost my github account it got hacked and marked as spam (3WEEKS no reply from github support)

### DISCUSSION THREAD

**Discussion by hugo88144** [community/community#143294] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i reset my passwords and  got 2fa an made ticket to github support but  now i cant even sign in anymore ,ive made a new account  but id like to know what i should to get my old account back  its was Hugo8814 , and email:hugopereira8814@gmail.com. thanks

**Comment by ebndev** [comment]
Hi @hugo88144,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
